.github/workflows/postgres.yml

```diff
@@ -0,0 +1,50 @@
+name: Postgres Versions Tests
+
+on:
+  schedule:
+    - cron: '0 18 * * 1'
+
+jobs:
+  postgres:
+    name: Python postgres unit tests
+    runs-on: ubuntu-18.04
+    strategy:
+      max-parallel: 5
+      matrix:
+        postgres-version: [9, 10, 11, 12, 13]
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres:${{ matrix.postgres-version }}
+        # Provide the password for postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6 for Postgres
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
```

Comment:
why not at least 3.8,  that's the current python version in Ubuntu LTS ?
It's also an intermediate between  current Debian stable (3.7) and this year to release stable (3.9)
3.6 seems quite outdated now

Reply:
I was just being conservative, it's still a supported release, and I found it unlikely that something that works on Py 3.6 would break in higher versions, but the other way around might do.

Reply:
yes, I understand it, I was selfishly thinking of making dev lives easier when testing in our local machines ;)